www/android/geolocation.js

```diff
@@ -29,7 +29,8 @@ var pluginToNativeWatchMap = {};
 
 module.exports = {
     getCurrentPosition: function (success, error, args) {
-        var win = function () {
+        var win = function (deviceApiLevel) {
+            if (deviceApiLevel === 31) args.enableHighAccuracy = true; // Workaround for bug specific to API 31 where requesting `enableHighAccuracy: false` results in TIMEOUT error.
```

Comment:
here `args` is optional and may be undefined. Therefore we need to check `args` first.